packages/manager/modules/pci/src/components/project/gateways/gateways-model-selector/template.html

```diff
@@ -15,20 +15,32 @@
         <oui-select-picker-section>
             <div
                 class="d-flex"
-                data-ng-if="gateway.blobs.technical.bandwidth.level"
+                data-ng-if="gateway.gatewayInfo[0].blobs || gateway.gatewayInfo[1].blobs"
```

Comment:
Why first or second one ? Maybe we should be more generic and just check any blob in gatewayInfo

Reply:
Taken care